blog/content/post/2020-12-14-trenchboot_attestation.md

```diff
@@ -0,0 +1,266 @@
+---
+title: 'Proof of concept implementation of RATS attestation for the TrenchBoot'
+abstract: 'This blog post will describe the concept of the IETF Remote
+          Attestation Procedures (RATS) and implementation of 
+          CHAllenge-Response based Remote Attestation (CHARRA) with TPM 2.0 for
+          TrenchBoot.'
+cover: /covers/trenchboot-logo.png
+author: norbert.kaminski
+layout: post
+published: true
+date: 2020-12-14
+archives: "2020"
+
+tags:
+  - TrenchBoot
+  - attestation
+  - TPM2
+  - firmware
+categories:
+  - Firmware
+  - Security
+
+---
+
+If you haven’t read previous blog posts from the TrenchBoot series, I strongly
+encourage you to catch up on them. The best way is to search under the
+[TrenchBoot tag](https://blog.3mdeb.com/tags/trenchboot/). This blog post will
+describe the concept of the IETF Remote Attestation Procedures (RATS) and
+implementation of CHAllenge-Response based Remote Attestation (CHARRA) with TPM
+2.0 for TrenchBoot.
+
+## Remote Attestation Procedures Architecture (RATS)
+
+The Remote Attestation Procedures (RATS) describe the methods of trustworthiness
```

Comment:
https://tpm2-software.github.io/2020/06/12/Remote-Attestation-With-tpm2-tools.html could be linked somewhere to learn about the basic terms used here

Reply:
I added this link with proper information to the summary.